src/components/ui/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Accordion, AccordionDetails, AccordionSummary } from './index';
+
+export default {
+  title: 'Ui/Accordion',
+  component: Accordion,
+  argTypes: {},
+} as ComponentMeta<typeof Accordion>;
+
+const Template: ComponentStory<typeof Accordion> = (args) => (
+  <Accordion {...args}>
+    <AccordionSummary>Foo</AccordionSummary>
```

Comment:
Let's add more meaningful names, for example Summary and Details

Reply:
Sorry I don't get you here. Summary and Details for the name of the story?

Reply:
@winston0410 No, instead of `Foo` and `Bar`

Reply:
@winston0410 I think this is missed